examples/README.md

```diff
@@ -29,6 +29,7 @@ Please note that these examples, although similar, are distinct from the ones us
 * [go](./go)
 * [java](./java)
 * [js](./js)
+* [next-js-netlify](./next-js-netlify)
```

Comment:
Make sure you add this to the [base Earthfile](examples/next-js-netlify/.gitignore) too.

Reply:
I'm leaving this off (at least for now). Both the +build and +deploy targets require supplying arguments. So this build would fail unless the user manually included args before executing either target.